src/features/web-compat.js

```diff
@@ -603,30 +603,38 @@ export default class WebCompat extends ContentFeature {
         // Chrome respects only the last viewport tag
         const viewportTag = viewportTags.length === 0 ? null : viewportTags[viewportTags.length - 1]
         const viewportContent = viewportTag?.getAttribute('content') || ''
-        const viewportContentParts = viewportContent ? viewportContent.split(/,|;/) : []
+        const viewportContentParts = viewportContent ? viewportContent.split(/\s*,\s*|\s*;\s*/) : []
```

Comment:
Does the trim not take care of this?

Reply:
It does, but the trimmed results end up in `parsedViewportContent`, and we reuse `viewportContentParts` to construct the final value

Reply:
I think I understand, to keep the whitespace around for later. Can you add this as a comment please :)

Reply:
I think trimming there would be clearer and simpler personally.